tests/suites/5_ee/01_vacuum/01_0002_ee_vacuum_drop_table.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+. "$CURDIR"/../../../shell_env.sh
+
+## Setup
+echo "drop database if exists test_vacuum_drop" | $MYSQL_CLIENT_CONNECT
+
+echo "CREATE DATABASE test_vacuum_drop" | $MYSQL_CLIENT_CONNECT
+echo "create table test_vacuum_drop.a(c int)" | $MYSQL_CLIENT_CONNECT
+
+echo "INSERT INTO test_vacuum_drop.a VALUES (1)" | $MYSQL_CLIENT_CONNECT
+
+echo "drop table test_vacuum_drop.a" | $MYSQL_CLIENT_CONNECT
+
+echo "vacuum drop table from test_vacuum_drop retain 0 hours" | $MYSQL_CLIENT_CONNECT
```

Comment:
How to `vacuum drop` all the databases which dropped sometime ago but don't know their names?

Reply:
well, i need to add an API to list dropped database within retention time first, after that i will come back for this pr, wait...